docker-compose.override.yml.dist

```diff
@@ -9,7 +9,9 @@ services:
       - '80:80'
       - '443:443'
     environment:
-      XDEBUG_REMOTE_ENABLED: 'false'
+      XDEBUG_REMOTE_ENABLED: 'true'
+      XDEBUG_REMOTE_PORT: '9000'
+      XDEBUG_REMOTE_AUTOSTART: 'true'
```

Comment:
what's the difference between ENABLED and AUTOSTART? if its enabled then it makes sense if it starts remote listening as well, no?

Reply:
Even if `XDEBUG_REMOTE_AUTOSTART` is `false` it will still listen on the `XDEBUG_REMOTE_PORT` but it won't trigger until you set the right cookie (or use one of the browser extensions to do it).

`XDEBUG_REMOTE_AUTOSTART` being `true` will connect to the remote machine on every request, including on CLI, which may not be desirable.